### ISSUE TITLE
DSM as input

### ISSUE THREAD

**Issue by Athelena** [tudelft3d/3dfier#128]
I think it might be useful to accept DSMs as input and not just point clouds. Now we know of several cases where people have to convert a DSM to a point cloud for use in 3dfier. So it might be nice to support it with 3dfier itself.

**Comment by liberostelios** [comment]
I like the issue, to give it more support!

**Comment by hugoledoux** [comment]
GDAL is already a dependency, so it should be feasible.

But 3dfier uses classification of the points, and the change ain't trivial. But not a gigantic task.

**Comment by liberostelios** [comment]
Isn't it possible that it's loaded with an assumed classification value of `0`?

Right now, we already do this: convert to a fake point-cloud with no classification at all.

**Comment by hugoledoux** [comment]
Yes I guess, but we need to think of the .yml file and consequences.

Just saying it's not a 1h task...
